pkg/infra/game/message/peer2peer.go

```diff
@@ -0,0 +1,19 @@
+package message
```

Comment:
Could you rename this file to `peer.go`, seems unnecessary to have numbers here.